packages/frontend/src/components/routes.jsx

```diff
@@ -0,0 +1,6 @@
+// @flow
+
+export const HomeRoute: String = '/';
```

Comment:
You should use the lowercase `string` type with flow.

Reply:
Okay, what is the difference?

Reply:
https://flow.org/en/docs/types/primitives/

Reply:
Thanks!